iOS/MasterTimeline/MasterTimelineViewController.swift

```diff
@@ -1052,7 +1052,7 @@ private extension MasterTimelineViewController {
 		let webFeed = coordinator.timelineFeed as! WebFeed
 		let homePageUrlTitle = NSLocalizedString("Copy Home Page URL", comment: "Copy Home Page URL")
 		let homePageCopyURLAction = UIAction(title: homePageUrlTitle,
-											 image: AppAssets.globeImage,
+											 image: AppAssets.safariImage,
```

Comment:
Shouldn't these icons match their direct equivalents on the Sidebar context menu?

Reply:
The do now, but I've removed the context menu work from this PR.